.github/workflows/ci-cd.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       # restore extension from the built-test job
       - uses: actions/download-artifact@v4
         with:
-          name: extension
+          name: extension-${{ matrix.node-version }}
```

Comment:
you will need to change it as well below for the vscode marketplace publish.

Reply:
oh - good catch